examples/connectivity/plot_extract_regions_statistical_maps.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
This example should be moved to the manipulating_visualizing section of the examples.